xhtml2pdf/xhtml2pdf_reportlab.py

```diff
@@ -32,10 +32,9 @@
 try:
     import StringIO
 except Exception:
-    from io import StringIO
-    StringIO_old = StringIO
+    from io import BytesIO
     class StringIO(object):
-        StringIO = StringIO_old
+        StringIO = BytesIO
```

Comment:
I think that here will be better to use:

    try:
        from io import BytesIO
    except ImportError:
        from StringIO import StringIO as BytesIO

And change everywhere in the code `StringIO.StringIO` with `BytesIO`. The `io` module is present even in Python 2.6. The except clause will be only for Python 2.5. Does xhtml2pdf support Python 2.5?